### PR TITLE
ColMarTech venders fixes

### LINKED ISSUE #1714: Tools Pouch (Full)

### LINKED ISSUE #1874: M280 Smartgunner Drum Belt

### LINKED ISSUE #1876: Electronics Pouch

### LINKED ISSUE #1965: Tools Pouch (Full)

### LINKED ISSUE #2267: 15 USCM Technical Satchel

### LINKED ISSUE #2268: 15 USCM Technical Chestrig

### LINKED ISSUE #2728: Uniform Vendor doesn't show pouch types
## Description

The ColMarTech surplus uniform vendor should show the type of the pouch in the UI. Like how the ColMarTech automated marine equipment rack shows it in parentheses.

**Reproduction**

**Screenshots**

SS14 Loader_Loo8Mdzgrg

**Additional context**

### PR DESCRIPTION
**Pull request by Tunguso4ka**
## About the PR

- Replaced and uncommented TODOs with IDs.
- Fixed wrong requisitions.yml name
- Closes #2728

Fixes #1714
Fixes #1874
Fixes #1876
Fixes #1965
Fixes #2267
Fixes #2268

### CODE REVIEW

**Review by DrSmugleaf — APPROVED** [review]
Thanks for the PR!

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/requisitions.yml
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/requisitions.yml:156]: Ok in retrospect I didn't add these in yet on purpose, since we don't have it implemented where it requires medical skill to use effectively. You can comment all uses of CMPouchMedkit or wait until I implement that to merge this.

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/requisitions.yml
Comment (Tunguso4ka) [reply Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/requisitions.yml]: done.

**Review by DrSmugleaf — APPROVED** [review]
Thanks for the PR!

**Pull request merged**
